user-service/src/main/java/rs/edu/raf/banka1/services/CardServiceImlp.java

```diff
@@ -0,0 +1,75 @@
+package rs.edu.raf.banka1.services;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import rs.edu.raf.banka1.model.Card;
+import rs.edu.raf.banka1.repositories.CardRepository;
+
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.List;
+
+@Service
+public class CardServiceImlp implements CardService{
+    @Autowired
+    private CardRepository cardRepository;
+
+    @Override
+    public List<Card> getAllCardsByAccountNumber(String accountNumber) {
+        return cardRepository.findByAccountNumber(accountNumber);
+    }
+
+    @Override
+    public Card createCard(String cardType, String cardName, String accountNumber, Integer limit) {
```

Comment:
cini mi se da je request sa fronta bio da hardkodujemo neke 2 kartice u trenutku pravljenja account-a.

Reply:
Napravio sam i to. Mada to nije neko preterano fino rešenje